variables.tf

```diff
@@ -92,6 +92,18 @@ variable "instance_count" {
   description = "The number of Tailscale Subnet Router EC2 instances you would like to deploy."
 }
 
+variable "monitoring_enabled" {
```

Comment:
We need to regenerate our README following this. Can you run `terraform-docs` locally to do so and commit the result?

Reply:
I installed and ran the pre-commit hooks that were listed here in `.pre-commit-config.yaml`. It updated the docs successfully, but it looks like the indentation alignment on the raw markdown changed.